proxy.h

```diff
@@ -13,6 +13,18 @@
 #include <type_traits>
 #include <utility>
 
+#if __has_cpp_attribute(msvc::no_unique_address)
+#define ___PRO_NO_UNIQUE_ADDRESS_ATTRIBUTE msvc::no_unique_address
+#elif __has_cpp_attribute(no_unique_address)
+#define ___PRO_NO_UNIQUE_ADDRESS_ATTRIBUTE no_unique_address
+#else
+#error "Proxy requires C++20 attribute no_unique_address"
```

Comment:
nit of nit: Maybe "the `no_unique_address` attribute"?

Reply:
I personally prefer to keep the current naming of macros